action/LedAnimation.action

```diff
@@ -0,0 +1,16 @@
+# Request
+# Supported Animation types
+int8 BLINK_LIGHTS = 1
```

Comment:
My thought is that people can just publish to the topic for Root's On/Off "animation" so I didn't include those here